src/wurf/git_checkout_resolver.py

```diff
@@ -41,20 +44,22 @@ def resolve(self):
         if self.git.current_commit(cwd=path) == self.checkout:
             return path
 
-        # Use the parent folder of the path retuned to store different
-        # versions of this repository
-        repo_folder = os.path.dirname(path)
-        checkout_path = os.path.join(repo_folder, self.checkout)
+        # Use the path retuned to create a unique location for this checkout
+        repo_hash = hashlib.sha1(path.encode('utf-8')).hexdigest()[:6]
```

Comment:
@mortenvp Can we get rid of these double hash codes? This seems really annoying for me.